.github/workflows/unit_test.yml

```diff
@@ -65,22 +65,7 @@ jobs:
       run: |
           set -x
           sudo apt-get update
-          # Use release 9 of llvm etc. - later versions have an unfixed
-          # bug on Ubuntu:
-          # https://github.com/iovisor/bcc/issues/2915
-          sudo apt-get -y install bison build-essential cmake flex git libelf-dev libfl-dev libedit-dev libllvm9 llvm-9-dev libclang-9-dev python zlib1g-dev
-          pushd /tmp
-          git clone --depth 1 --branch v0.20.0 https://github.com/iovisor/bcc.git
-          mkdir -p bcc/build; cd bcc/build
-          # Symlink /usr/lib/llvm to avoid "Unable to find clang libraries"
-          # The directory appears only to be created when installing the
-          # virtual llvm-dev package.
-          # https://github.com/iovisor/bcc/issues/492
-          sudo ln -s /usr/lib/llvm-9 /usr/local/llvm
-          cmake ..
-          make
-          sudo make install
-          popd
+          sudo apt-get install -y libbpfcc-dev linux-headers-$(uname -r)
```

Comment:
libbpfcc-dev is not updated, it stays at 0.12 :(

Reply:
yea... I found it.